src/query/query_executor.c

```diff
@@ -15699,8 +15701,10 @@ qexec_execute_mainblock_internal (THREAD_ENTRY * thread_p, XASL_NODE * xasl, XAS
 	    }
 	}
 
-      /* monitor */
-      perfmon_inc_stat (thread_p, PSTAT_QM_NUM_SELECTS);
+      if (xasl->type != MERGELIST_PROC)
```

Comment:
```suggestion
      if (xasl->type == BUILDLIST_PROC || xasl->type == BUILDVALUE_PROC)
```
사실 정의상으로는 buildlist, buildvalue를 제외한 나머지 proc은 select가 아니므로 이렇게 처리하는 것이 맞아 보입니다.

Reply:
다만, update/delete가 buildlist proc 등으로 처리되기 때문에, 이 경우 upddel cls 등으로 확인하여 제외하는 것도 좋을 것 같습니다.

Reply:
[5cdfda7](https://github.com/CUBRID/cubrid/pull/6582/commits/5cdfda7a50792677c18cfab2b78f6ab72d845a75)에서 수정했습니다.